crates/cheatable-starknet/src/cheatcodes/declare.rs

```diff
@@ -33,38 +31,40 @@ use starknet::core::types::contract::CompiledClass;
 impl CheatedState {
```

Comment:
Would it be possible to put BlockifierState into CheatedState to abstract the separation for forge devs? `CheatedState` could be renamed to `CheatnetState`

Reply:
Seems like a valid take to me, @Radinyn WDYT?